internal/engine/filter/convert.go

```diff
@@ -252,6 +258,14 @@ func escape(a string) string {
 	return a
 }
 
+func isQuoted(a string) bool {
+	dble := regexp.MustCompile(`"`)
```

Comment:
I think it's a lot easier to just verify if there's any double quotes in the string. No need for regular expressions:

```
strings.Contains(a, `"`)
```

See https://pkg.go.dev/strings#Contains.

Also, single quotes are fine, so we shouldn't error out because of them.